src/Components/Subscriptions/FormSubscription/Index.jsx

```diff
@@ -45,7 +56,7 @@ const FormSubscription = () => {
   const subscriptionUpdated = {
     classId: data.classId?._id,
     members: data.members?.map((member) => member._id),
-    date: data.date && new Date(data.date).toISOString().substr(0, 10)
```

Comment:
Don't delete `&& new Date(data.date).toISOString().substr(0, 10)`.
![image](https://github.com/BaSP-m2023/nico-megarocket-app/assets/102418096/a77f1d3d-c6f3-4161-87d7-7f9b035dd7c4)


 This brings the actual date to the Date Input . Without it it shows like on the photo above.

Reply:
Ok!